sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -302,75 +120,11 @@ SettingSelector selector = new SettingSelector().setKeyFilter(key + "," + key2);
-
-<!-- embedme ./src/samples/java/com/azure/data/appconfiguration/ReadmeSamples.java#L151-L152 -->
-```Java
-configurationClient.setConfigurationSetting("some_key", "some_label", "some_value");
-ConfigurationSetting setting = configurationClient.setReadOnly("some_key", "some_label", true);
-```
-### Clear read only from a Configuration Setting
-
-Clear read-only from a configuration setting.
-
-<!-- embedme ./src/samples/java/com/azure/data/appconfiguration/ReadmeSamples.java#L156-L156 -->
-```Java
-ConfigurationSetting setting = configurationClient.setReadOnly("some_key", "some_label", false);
-```
-
-## Troubleshooting
-
-### General
-
-When you interact with App Configuration using this Java client library, errors returned by the service correspond to the same HTTP status codes returned for [REST API][rest_api] requests. For example, if you try to retrieve a configuration setting that doesn't exist in your configuration store, a `404` error is returned, indicating `Not Found`.
-
-App Configuration provides a way to define customized headers through `Context` object in the public API. 
-
-<!-- embedme ./src/samples/java/com/azure/data/appconfiguration/ReadmeSamples.java#L160-L169 -->
-```java
-// Add your headers
-HttpHeaders headers = new HttpHeaders();
-headers.put("my-header1", "my-header1-value");
-headers.put("my-header2", "my-header2-value");
-headers.put("my-header3", "my-header3-value");
-// Call API by passing headers in Context.
-configurationClient.addConfigurationSettingWithResponse(
-    new ConfigurationSetting().setKey("key").setValue("value"),
-    new Context(AddHeadersFromContextPolicy.AZURE_REQUEST_HTTP_HEADERS_KEY, headers));
-// Above three HttpHeader will be added in outgoing HttpRequest.
-```
-For more detail information, check out the [AddHeadersFromContextPolicy][add_headers_from_context_policy]
-
-### Default HTTP Client
-All client libraries by default use the Netty HTTP client. Adding the above dependency will automatically configure 
-the client library to use the Netty HTTP client. Configuring or changing the HTTP client is detailed in the
-[HTTP clients wiki](https://github.com/Azure/azure-sdk-for-java/wiki/HTTP-clients).
-
-### Default SSL library
-All client libraries, by default, use the Tomcat-native Boring SSL library to enable native-level performance for SSL 
-operations. The Boring SSL library is an uber jar containing native libraries for Linux / macOS / Windows, and provides 
-better performance compared to the default SSL implementation within the JDK. For more information, including how to 
-reduce the dependency size, refer to the [performance tuning][performance_tuning] section of the wiki.
-
 ## Next steps
 
 - Samples are explained in detail [here][samples_readme].
 - [Quickstart: Create a Java Spring app with App Configuration][spring_quickstart]
+- The [Azure SDK for Java wiki][wiki] contains guidance on logging, performance tuning, identity / authentication, and much more.
```

Comment:
We should start cleaning up our wiki if we are going to put this link in all our readmes. We have a few dead links in our wiki today. I will go through the links and create GitHub issues to update the wiki.